File: rest/call/list-get-example-4/list-get-example-4.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

@client.calls.list(status: 'completed',
                   'start_time>': '2009-07-04',
                   'start_time<': '2009-07-06').each do |call|
  puts call.to
end

File: rest/call/list-get-example-4/list-get-example-4.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
        TwilioClient.Init(accountSid, authToken);

        var calls = CallResource.Read(status: CallResource.StatusEnum.InProgress,
                                      startTimeAfter: new DateTime(2009, 07, 04),
                                      endTimeBefore: new DateTime(2009, 07, 06));

        foreach (var call in calls)
        {
            Console.WriteLine(call.To);
        }
    }
}
